pokedex-with-bit/flows/index.yml

```diff
@@ -1,30 +1,33 @@
-!Flow
-version: '1'
-pods:
-  - name: crafter
-    show_exc_info: true
-    uses: pods/craft.yml
-    shards: $JINA_SHARDS
-    read_only: true
-  - name: encoder
-    show_exc_info: true
-    uses: pods/encode.yml
-    shards: $JINA_SHARDS
-    timeout_ready: 600000
-    read_only: true
-  - name: vec_idx
-    show_exc_info: true
-    uses: pods/vec.yml
-    shards: $JINA_SHARDS_INDEXERS
-    polling: any
-    timeout_ready: 100000 # larger timeout as in query time will read all the data
-  - name: doc_idx
-    uses: pods/doc.yml
-    shards: $JINA_SHARDS_INDEXERS
-    polling: any
-    timeout_ready: 100000 # larger timeout as in query time will read all the data
-    needs: [gateway]
-  - name: join_all
-    uses: _merge
-    needs: [doc_idx, vec_idx]
-    read_only: true
+!Flow                                       # This file defines the indexing flow for this example
+version: '1'                                # Version number you can choose freely
+with:                                       # The with keyword defines parameters for the flow
+  workspace: "workspace"                    # Set the workspace path
+  py_modules:                               # Define the location of the executors implementation
+    - executors.py
+executors:                                  # List of executors for this flow
+  - name: crafter                           # Arbitrary name of the first executor
```

Comment:
Arbitrary name is not so good I think.

Reply:
Can just go with "Name"

Reply:
https://github.com/jina-ai/examples/pull/626/commits/473cafec51508030ab3c81c8bfa4cc08439a92ec